### ISSUE TITLE
Make About page in main app

### ISSUE THREAD

**Issue by micahdbak** [CSSS/csss-site-frontend#44]

**Comment by micahdbak** [comment]
@EarthenSky - you still want to do this?

**Comment by EarthenSky** [comment]
Not really, but I can do it if necessary. As before, backend & elections pages are priority atm.

If someone else is interested please attempt!

**Comment by micahdbak** [comment]
TODO for @micahdbak add more info about what this page should contain
